output.tf

```
@@ -1,3 +1,22 @@
+output "address" {
+  value = aws_db_instance.mz_rds_demo_db.address
+}
+output "password" {
+  sensitive = true
+  value     = aws_db_instance.mz_rds_demo_db.password
+}
+output "username" {
+  value = aws_db_instance.mz_rds_demo_db.username
+}
+output "db_name" {
+  value = aws_db_instance.mz_rds_demo_db.db_name
+}
+
```

What we can do is create a unified output as follows:

```suggestion
output "rds_details" {
  value = aws_db_instance.mz_rds_demo_db
}
```

That way when someone uses the module with:

```hcl
module "rds_postgres" {
  source  = "MaterializeInc/rds-postgres/aws"
  version = "0.1.1"
  # insert the 2 required variables here
}
```

They will be able to reference the `rds_details` output with:

```
module.rds_postgres.rds_details
```

And they will have access to the complete object.